### ISSUE TITLE
Packaging for Beanstalk deployment fails with certain Windows 10 Regional Formats

### ISSUE THREAD

**Issue by tofalck** [aws/aws-toolkit-visual-studio#82]
**Describe the bug**
If the "Regional format" is not set to "English (United States)", the AWS Toolkit for Visual Studio breaks when deploying an Asp.Net Core project to Beanstalk with the error in the Amazon Web Services build output toolbox window: 

```
..zipping publishing directory
...caught exception during deployment package creation - 1 is not a supported code page.
```

**To Reproduce**
1. Close VS 
2. Go to System settings -> Language Settings -> Regional format dropdown -> I picked "English (Denmark)"
3. Open Asp.Net Core solution
4. Right click project in solution explorer
5. Choose "Publish to AWS Elastic Beanstalk..."
6. Create new or redeploy and follow the rest of the wizard, Deploy at the end
7. Prompt is shown and the build breaks with the above error
8. Run step 1-6 again, but in step 2 set the Regional format to "English (United States)" -> step 6 yields no errors

**Expected behavior**
- Users should be able to have their local settings as they see fit, so the toolkit should either set the codepage or allow the developer to override the system settings through configuration.

**Screenshots**

![image](https://user-images.githubusercontent.com/2613099/80577988-1a124d80-8a08-11ea-9a7a-9667ce1f93a0.png)

**Environment:**
- Windows 10 Pro
- Microsoft Visual Studio Community 2019, Version 16.5.4
- AWS Toolkit for VS 2017 and 2019, Version 1.17.1.0
- Asp.Net Core 3.1 project
- Beanstalk: Windows Core 2019 2.5.2. with IIS 10

**Comment by awschristou** [comment]
Hi @tofalck thank you for the excellent repro steps.

This problem is [related to the zip library we're using](https://github.com/icsharpcode/SharpZipLib/issues/195). We'll investigate this further.

In the meantime, there is no in-toolkit workaround for this. Changing the system region settings as you mention helps, but is not ideal.

**Comment by awschristou** [comment]
A fix for this has been released with v1.18.1.0 of the Toolkit.
